packages/client/src/generation/utils/buildInjectableEdgeEnv.ts

```diff
@@ -1,6 +1,6 @@
 import type { InternalDatasource } from '../../runtime/utils/printDatasources'
```

Comment:
Here I simplified the code and updated the comments, hopefully more comprehensive.

Reply:
(this code is only for the edge client)

Reply:
(this code is only for the edge client)